src/app/components/Modals/FeedbackModal.tsx

```diff
@@ -142,7 +142,7 @@ export const FeedbackModal: React.FC<FeedbackModalProps> = ({
         </h3>
         <button
           className="btn btn-circle btn-ghost btn-sm"
-          onClick={() => onClose(false)}
```

Comment:
I remember we had to use such flags, @jeremy-babylonlabs maybe you remember better
Is it okay to remove them now?